sig-scalability/annual-report-2021.md

```diff
@@ -33,9 +33,9 @@ There isn't any formal process. Updates are given during regular SIG meetings.
 
 **When was your last monthly community-wide update? (provide link to deck and/or recording)**
 
-January'20: [slide deck]
+August'20: [slide deck]
```

Comment:
I knew I was missing somthing but I couldn't find these slides...

How about putting both here instead?

Reply:
Initially, I wanted to do that, but then I read the heading - "last ... community-wide update" (btw, no idea why there is monthly there). 
But I'm fine with having both as well. You decide :)

Reply:
@mrbobbytables - thoughts? [I would keep both, because they were both in 2020]

Reply:
I'd lean towards both, it's still useful information. :)

Reply:
Done. Sorry, I completely forgot about it.